src/raylib.h

```diff
@@ -1581,6 +1581,15 @@ RLAPI void DetachAudioStreamProcessor(AudioStream stream, AudioCallback processo
 RLAPI void AttachAudioMixedProcessor(AudioCallback processor); // Attach audio stream processor to the entire audio pipeline
 RLAPI void DetachAudioMixedProcessor(AudioCallback processor); // Detach audio stream processor from the entire audio pipeline
 
+#if defined(SUPPORT_EVENTS_AUTOMATION)
+//------------------------------------------------------------------------------------
+// Event Recording & Replaying
+//------------------------------------------------------------------------------------
+RLAPI void BeginEventRecording(bool displayText);
```

Comment:
I prefer to avoid this kind of parameters, text display can be configured with a compilation flag. Due to the advance nature of the API there shouldn't be a problem for adavance users.